pkg/apis/pipeline/v1alpha1/pipelinerun_validation.go

```diff
@@ -62,5 +62,12 @@ func (ps *PipelineRunSpec) Validate(ctx context.Context) *apis.FieldError {
 		}
 	}
 
+	if ps.ExpirationSecondsTTL != nil {
+		// DelayDeleteTTL should be a valid duration of at least 0.
```

Comment:
DelayDeleteTTL ?

Reply:
Sorry, this is a outdated field, and I have updated it with ExpirationSecondsTTL in https://github.com/tektoncd/pipeline/pull/1486